components/SpotDetails/SpotSocial/SpotSocialComment.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export default function SpotSocialComment({ text }) {
+  return (
+    <>
+      <StyledEntry>
+        <StyledText>{text}</StyledText>
+      </StyledEntry>
+    </>
```

Comment:
In this case you don't need a Fragment `<>` cause your StyledEntry `<StyledEntry>` is already the top tag:

```suggestion
      <StyledEntry>
        <StyledText>{text}</StyledText>
      </StyledEntry>
```